erb/docs.rb

```diff
@@ -0,0 +1,4 @@
+require 'erb'
+template = File.read('erb/README.erb')
+renderer = ERB.new(template, nil, '-')
+File.write('README.md', renderer.result())
```

Comment:
@aarongorka in case you're wondering, all this Ruby supports only the code generation in the docs and it would be moved into Cookiecutter once I figure out where that is and how it works. Also, in this case, this code really does need to be written in Ruby- it's just not possible to do what I'm doing here using Python/Jinja2 - I [tried](https://alexharv074.github.io/2018/11/09/generating-markdown-from-source-code-in-erb.html).